include/deal.II/particles/particle_accessor.h

```diff
@@ -416,10 +409,9 @@ namespace Particles
 
   template <int dim, int spacedim>
   inline ParticleAccessor<dim, spacedim>::ParticleAccessor()
-    : particles(nullptr)
+    : particles_on_cell(typename particle_container::iterator())
```

Comment:
This is like initializing an object of type `X` by copying an `X()`. It's easier to just call this instead :-)
```suggestion
    : particles_on_cell()
```
or if you absolutely must,
```suggestion
    : particles_on_cell({})
```